packages/example-rpc-server/package.json

```diff
@@ -0,0 +1,62 @@
+{
+  "name": "@loopback/example-rpc-server",
+  "version": "4.0.0-alpha.0",
+  "description": "A basic RPC server using a made-up protocol.",
+  "keywords": [
+    "loopback-application",
+    "loopback"
+  ],
+  "main": "./dist/src/index.js",
```

Comment:
Shouldn't this point to `index.js`?

Reply:
Actually it appears that `index.d.ts` and `index.js` are missing in this example yet it supports Node 6 so I'm actually confused as to how this even passes on 6 rn.